applications/salk-portal/src/components/ExplorerSidebarDialog.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import {
+  Box,
+  makeStyles,
+  Grid,
+  Typography,
+  Link,
+  Container
+} from "@material-ui/core";
+import { switchActiveColor, headerBorderColor } from "../theme";
+import LOGO from "../assets/images/logo_icon.svg"
+import Modal from "./common/BaseDialog";
+
+const useStyles = makeStyles(() => ({
+  learnMore: {
+    '& .details': {
+      flexGrow: 1,
+
+      '& .detail-block + .detail-block': {
+        marginTop: '1.5rem',
+      },
+
+      '& .logo-container': {
+
+        '& .logo': {
+          display:"flex",
+          justifyContent:"center",
+          alignItems:"center"
+        },
+
+        '& .MuiContainer-root': {
+            border: "0.063rem solid",
+            width: "5rem",
+            height: "5rem",
+            borderRadius: "50%",
+            color:headerBorderColor
+        }
+      }, 
+
+  
+      '& .MuiTypography-root': {
+        fontSize: '0.75rem',
+        lineHeight: '0.9375rem',
+        fontWeight: 600,
+        marginBottom: '0.5rem',
+      },
+
+      '& .MuiLink-root': {
+        color:switchActiveColor,
+        fontWeight: 400,
+      },
+
+      '& p': {
+        '&.MuiTypography-root': {
+          fontWeight: 400,
+        },
+      },
+    },
+  },
+
+}));
+
+
+
+const ExplorerDialog = (props: any) => {
+  const classes = useStyles();
+  const { open, handleClose} = props;
+  return (
+    <Modal open={Boolean(open)} handleClose={handleClose} title="About Mouse Coord Atlas" maxWidth="xs">
+      <Box className={classes.learnMore}>
```

Comment:
@paularah so essentially here you are making two parent elements first is Box and second is Grid container. Keep one parent only may be Grid container cause rest of the design is using Grid items